### PR TITLE
scikit SVM train and prediction

### PR DESCRIPTION
**Pull request by bogdanlupu**

### CODE REVIEW

**Review by stefan-matcovici — APPROVED** [review]

File: data/p3/p3.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+from sklearn.svm import SVC
+from sklearn.datasets import load_svmlight_file
+
+from utils import save_plot
+
+
+def load_data(path):
+    with open(path, 'rb') as f:
+        return load_svmlight_file(f, n_features=784)
```
Comment (stefan-matcovici) [inline data/p3/p3.py]: this load returns a scipy.sparse.csr_matrix object, you don't want to train svm with objects
see https://docs.scipy.org/doc/scipy-0.19.0/reference/generated/scipy.sparse.csr_matrix.todense.html

File: data/p3/p3.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    global c_values
+    train_x, train_y = load_data(train_file)
+    test_x, test_y = load_data('test-01-images.svm')
+    test_y = numpy.array(test_y)
+    c_values = [0.01, 0.1, 1, 10, 100]
+    training_errors = []
+    testing_errors = []
+    for c in c_values:
+        model = train(train_x, train_y, c)
+        predict_y = numpy.array(model.predict(test_x))
```
Comment (stefan-matcovici) [inline data/p3/p3.py]: for accuracy you could use just model.score
